docs/03_server/05_access-control/03_password_authentication.md

```diff
@@ -387,23 +424,47 @@ The error codes that can be returned are currently:
+- `INSUFFICIENT_CHARACTERISTICS` - can be provided if you have configured passwords to be successful only if a predetermined set of strength checks pass. Should be provided alongside "real" error codes.
 - `ILLEGAL_SEQUENCE` - Numeric/alphabetic sequence detected.
 
-### Reset password
-This can only be called by an administrator; it simply specifies a user name and sets the password to blank.
+### Logout
 
-#### Reset request
-    MESSAGE_TYPE = EVENT_RESET_USER_PASSWORD
-    DETAILS.USER_NAME = JohnWolf
+As explained in the previous section, if the response for the login attempt is `MAX_ACTIVE_SESSIONS_REACHED`, the server will reply with a list of active sessions, so the client can optionally terminate one of them. In this scenario, the client would need to send a LOGOUT message with the specific SESSION_ID value of the session to terminate.
 
-#### Reset response
-    MESSAGE_TYPE = EVENT_RESET_USER_PASSWORD_ACK
+The response message in that case would look like this:
+
+```
+MESSAGE_TYPE = EVENT_LOGIN_AUTH_NACK
+ERROR[0].@type = LoginError
+ERROR[0].CODE = MAX_ACTIVE_SESSIONS_REACHED
+ERROR[0].TEXT = Problem logging in
+ERROR[0].STATUS_CODE = 403 Forbidden
+ERROR[0].DETAILS.SESSION[0].SESSION_ID = abb8f6be-8009-4370-a474-3a0ded4dc2cf
+ERROR[0].DETAILS.SESSION[0].HOST = host1
+ERROR[0].DETAILS.SESSION[0].LAST_ACCESS_TIME = 2024-01-25 14:27:54.925 (1706192874925)
+ERROR[0].DETAILS.SESSION[1].SESSION_ID = c8c04ba5-d450-48f7-a863-2a500fe0a4e7
+ERROR[0].DETAILS.SESSION[1].HOST = host2
+ERROR[0].DETAILS.SESSION[1].LAST_ACCESS_TIME = 2024-01-25 14:27:55.037 (1706192875037)
+ERROR[0].DETAILS.SESSION[2].SESSION_ID = f6255056-952f-4985-b984-6e48c822c3a4
+ERROR[0].DETAILS.SESSION[2].HOST = host3
+ERROR[0].DETAILS.SESSION[2].LAST_ACCESS_TIME = 2024-01-25 14:27:55.235 (1706192875235)
+ERROR[0].DETAILS.SESSION[3].SESSION_ID = 29220a26-0753-4408-9c28-9f00457e98ac
+ERROR[0].DETAILS.SESSION[3].HOST = host4
+ERROR[0].DETAILS.SESSION[3].LAST_ACCESS_TIME = 2024-01-25 14:27:55.438 (1706192875438)
+```
+
+A logout message to terminate the first session would look like this:
+
+```
+MESSAGE_TYPE = EVENT_LOGOUT
+DETAILS.USER_NAME = JohnWolf
+DETAILS.SESSION_ID = abb8f6be-8009-4370-a474-3a0ded4dc2cf
+```
 
 ## Post-authentication
-Once the user has been authenticated, the server expects heartbeat messages, as defined in the interval setting on the ACK message. If the GUI misses a configurable number of heartbeats, the session will automatically expire. In response to a heartbeat, the GUI will receive a list of available services and their details.
+Once the user has been authenticated, the server can optionally receive heartbeat messages, as defined in the interval setting on the ACK message. In response to a heartbeat, the GUI will receive a list of available services and their details, which could include multiple hostnames if the environment is configured to not use Consul as the cluster mode.
```

Comment:
This only applies if you are not building a website (websites can only target one host) and if the backend is not using consul as a cluster mode, so I had to clarify the explanation.

Reply:
I have now changed the heading and moved this to the end of the post-auth section, to ensure that it is not a BAU consideration

Reply:
@jacinpoz please check lines 563-588